src/systems/systems/src/electrical/battery_charge_limiter.rs

```diff
@@ -1015,10 +1038,7 @@ mod tests {
                 .battery_push_button_off()
                 .run(Duration::from_millis(
                     OpenContactorObserver::BATTERY_CHARGING_CLOSE_DELAY_MILLISECONDS,
-                ))
-                .then_continue_with()
-                .battery_push_button_auto()
-                .run(Duration::from_millis(0));
```

Comment:
Clearly the name of the function specifies the push button is off: `contactor_not_closed_when_battery_voltage_below_charge_threshold_and_battery_bus_above_threshold_for_greater_than_225ms_with_push_button_off`

I'm not sure why I set it to auto here. It started to fail with these changes (which makes sense, as it would connect to the bus when set to auto). Thus removed this code here.